### DISCUSSION TITLE
Patch based augmentation

### DISCUSSION THREAD

**Discussion by Paddy-Xu** [TorchIO-project/torchio#952] [Ideas]
I think those augmentations are very useful, but as far as I understand, those are based on the whole image
Is it possible to do patch based augmentations?

**Comment by maxwellreynolds** [comment]
Also curious about this. The transformation could be applied on the patch, but is it better practice to apply on the entire image then crop? The former seems like it would be more efficient for slower transformations.

**Comment by romainVala** [comment]
Hello
this question was discuss there #954
